testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js

```diff
@@ -1114,6 +1114,39 @@ QUnit.module('Lookup', {
         assert.equal(this.popup.option('height'), popupHeight, 'popupHeight applied to popover');
     });
 
+    [true, false].forEach((usePopover) => {
+        QUnit.test(`overlay should contain role="dialog" when usePopover: ${usePopover}`, function(assert) {
+            const instance = $('#lookup').dxLookup({
+                usePopover: usePopover
+            }).dxLookup('instance');
+
+            this.togglePopup();
+
+            const $overlayContent = instance._popup.$overlayContent();
```

Comment:
Let's try to not use a private api in tests. If we change it in the future these tests become red.
`this.popup.content().parent()`